support-e2e/tests/tieredCheckout.test.ts

```diff
@@ -182,3 +182,68 @@ test.describe('Subscribe (S+) incl PromoCode via the Tiered checkout', () => {
 		});
 	});
 });
+
+const thresholdTests = [
+	{ amount: '10', contributionType: 'monthly', product: 'All-access digital' },
+	{ amount: '9', contributionType: 'monthly', product: 'Support' },
+	{ amount: '95', contributionType: 'annual', product: 'All-access digital' },
+	{ amount: '94', contributionType: 'annual', product: 'Support' },
+	{
+		amount: '10',
+		contributionType: 'monthly',
+		promoCode: 'E2E_TEST_SPLUS_MONTHLY',
+		product: 'All-access digital',
+	},
+	{
+		amount: '9',
+		contributionType: 'monthly',
+		promoCode: 'E2E_TEST_SPLUS_MONTHLY',
+		product: 'Support',
+	},
+	{
+		amount: '95',
+		contributionType: 'annual',
+		promoCode: 'E2E_TEST_SPLUS_ANNUAL',
+		product: 'All-access digital',
+	},
+	{
+		amount: '94',
+		contributionType: 'annual',
+		promoCode: 'E2E_TEST_SPLUS_ANNUAL',
+		product: 'Support',
+	},
+];
+test.describe('Thresholds - the product is "Support" or "All-access digital" correctly based on selected-amount', () => {
+	for (const testConfig of thresholdTests) {
+		test(`Thresholds - ${testConfig.product} ${testConfig.amount}/${
+			testConfig.contributionType
+		}/${testConfig.promoCode ?? ''}`, async ({ context, baseURL }) => {
+			const page = await context.newPage();
+			const urlParams = new URLSearchParams({
+				'selected-amount': testConfig.amount,
+				'selected-contribution-type': testConfig.contributionType,
+			});
+			if (testConfig.promoCode) {
+				urlParams.append('promoCode', testConfig.promoCode);
+			}
+			await setupPage(
+				page,
+				context,
+				baseURL,
+				`/uk/contribute/checkout?${urlParams.toString()}`,
+			);
+
+			const orderSummaryHeading = page.getByRole('heading', {
+				name: 'Your subscription',
+			});
+			const orderSummarySection = page
+				.locator('section')
+				.filter({ has: orderSummaryHeading })
+				.nth(1);
+
+			expect(
```

Comment:
Should we be checking the `total` amount is as expected too? Tests 5/8 look wrong on laywright but perhaps the total fix from previous playwright tests needs to go in first?

Reply:
fyi: updated and tests 5/8 are fine

Reply:
I don't really want to be checking the total here (this is done with the promoCode and checkout tests. 

This is just saying "Am I showing you the right product given our thresholds".

Reply:
You're right - the branch needed updating with `main`.